ResourceStatusSystem/scripts/dirac-rss-sync.py

```diff
@@ -94,25 +100,51 @@ def synchronize():
   from DIRAC.ResourceStatusSystem.Utilities      import Synchronizer
 
   synchronizer = Synchronizer.Synchronizer()
-  
+
   if switchDict[ 'element' ] in ( 'Site', 'all' ):
     subLogger.info( 'Synchronizing Sites' )
     res = synchronizer._syncSites()
     if not res[ 'OK' ]:
       return res
-    
+
   if switchDict[ 'element' ] in ( 'Resource', 'all' ):
     subLogger.info( 'Synchronizing Resource' )
     res = synchronizer._syncResources()
     if not res[ 'OK' ]:
       return res
-  
+
   if switchDict[ 'element' ] in ( 'Node', 'all' ):
     subLogger.info( 'Synchronizing Nodes' )
     res = synchronizer._syncNodes()
     if not res[ 'OK' ]:
       return res
-  
+
+  return S_OK()
+
+def initSites():
+  '''
+    Initializes Sites statuses taking their values from the "SiteMask" table of "JobDB" database.
+  '''
+
+  jobClient = JobDB()
+  rssClient = ResourceStatusDB()
+
+  sites = jobClient.getAllSiteMaskStatus()
+
+  if not sites[ 'OK' ]:
+    subLogger.error( sites[ 'Message' ] )
```

Comment:
you have to exit if you have the error, otherwise you get an exception the line after